js/src/services/vue.service.js

```diff
@@ -0,0 +1,103 @@
+import Vue from 'vue';
+import atkInlineEdit from '../components/inline-edit.component';
+import itemSearch from '../components/item-search.component';
+
+
+let atkComponents = {
+  'atk-inline-edit' : atkInlineEdit,
+  'atk-item-search' : itemSearch,
```

Comment:
does this have to be populated for all new components? does not seem very universal.

Reply:
Vue instance need to have it's component register when you created it. This is one way of doing it and it is ok for component that atk managed. For external component, there would be another way for registering them like the createVue method in Vue service.

Reply:
ok

Reply:
at some point if we have a vue create guide, should mention that then ;)